src/reducers/ticketReducers.js

```diff
@@ -45,6 +45,18 @@ const addIncidentsToState = (state, incidents) => {
   return newState
 }
 
+const removeTicketFromState = (state, ticketId) => {
+  let newState = { ...state }
+  newState[ticketId] = null
+  return newState
+}
+
+const removeAllTicketsFromState = (state, ticketIds) => {
```

Comment:
It seems strange that this action and method are named 'removeAll' when they remove a given range rather than all the tickets.

return Object.Assign({}, ...Object.keys(state).map(key => ({key: null})) )

I think this (or something similar) will remove all tickets from state. For a simpler change, try using Object.keys(state) as ticketIds rather than passing it in.